### PR TITLE
Add more test coverage

### PR DESCRIPTION
**Pull request by martijn00**
## Connection with issue(s)

Close #???

Connected to #???

## To Do

- [ ] Read [contributing guide](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/CONTRIBUTING.md)
- [ ] Check the original issue to confirm it is fully satisfied
- [ ] Add solution description to help guide reviewers
- [ ] Add unit test to verify new or fixed behaviour
- [ ] If apply, add documentation to code properties and package readme

### CODE REVIEW

**Review by deandreamatias — COMMENTED** [review]
Make sense that core validators/helpers use BaseValidator if not use translatedErrorText?
```dart
String get translatedErrorText => '';
  // coverage:ignore-end
  ```

**Comment by martijn00** [comment]
Yes it still makes sense because they use the validate methods and other users can extend them.

**Comment by deandreamatias** [comment]
> Yes it still makes sense because they use the validate methods and other users can extend them.

Yeah, but maybe can create a CoreValidator without translatedErrorText and BaseValidator extends this? I don't if this could be work.

Ignore a thing that won't be used, I don't see like a good practice

**Comment by martijn00** [comment]
@deandreamatias I've implemented this now. Can you check and merge?

File: README.md [hunk 1]
```diff
@@ -382,7 +382,7 @@ We welcome efforts to internationalize/localize the package by translating the d
 #### Add new validator
 
 1. Add a new validator to one of the folders in the `src` folder.
-2. Implement it using the `BaseValidator` class. Override the `validateValue` method and let the base class handle the null check in the `validate` method.
+2. Implement it using the `BaseValidator` or `TranslatedValidator` class. Override the `validateValue` method and let the base class handle the null check in the `validate` method. When using a translation also override `translatedErrorText`.
 3. Override the `translatedErrorText` property and return the correct translation from `FormBuilderLocalizations.current.`.
```
Comment (deandreamatias) [inline README.md:386]:
```suggestion
2. Implement it using the `BaseValidator` or `TranslatedValidator` class. Override the `validateValue` method and let the base class handle the null check in the `validate` method.
3. When using a `TranslatedValidator, Override the `translatedErrorText` property and return the correct translation from `FormBuilderLocalizations.current.`.
```

**Review by deandreamatias — APPROVED** [review]

**Pull request merged**